src/parser/scanner.lex

```diff
@@ -285,8 +285,11 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
                                         yyterminate();
                                     }
                                 }
-                                int64_t val = 0;
+                                uint64_t val = 0;
```

Comment:
Hex and octal do not need to be processed

Reply:
why？ Don't they overflow?